### PR TITLE
docs(wechat-native-connector): readme (zh-CN)

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

readme zh-CN version

## Testing
[Preview](https://github.com/logto-io/logto/tree/gao-wechat-native-readme-zh-cn/packages/connector-wechat-native/README.md)

### CODE REVIEW

File: packages/connector-wechat-native/README.md [hunk 1]
```diff
@@ -235,4 +263,159 @@ Once WeChat native connector is enabled, you can build and run your app to see i
...
+
+**iOS 应用**
+
+勾选「iOS 应用」与你的应用的目标平台。
+
+![App platform](/packages/connector-wechat-native/docs/platform.png)
+
+如果你在「应用已上架」中选择了「否」，你可以在这一步跳过「AppStore 下载地址」的填写。
+
+填写 _Bundle ID_、_测试版本 Bundle ID_ 和 _Universal Links_（实际只需要一个 link 😂）。
```
Comment (xiaoyijun) [inline packages/connector-wechat-native/README.md:316]: 实际上只需要填写同一个 link?

File: packages/connector-wechat-native/README.md [hunk 2]
```diff
@@ -235,4 +263,159 @@ Once WeChat native connector is enabled, you can build and run your app to see i
 
 # 微信原生连接器
 
+## 开始上手
+
+如果你还不知道连接器的概念，或者还没有将本连接器添加至你的「登录体验」，请先参见 [Logto 教程](https://docs.logto.io/zh-cn/docs/tutorials/get-started/enable-social-sign-in)。
+
+在原生应用中，你无法将 web 用于微信登录。必须跳转至微信 app，并使用他们提供的 SDK 才能完成。
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/connector-wechat-native/README.md:270]: yes we cannot

File: packages/connector-wechat-native/README.md (hunk 1 above)
Comment (gao-sun) [reply packages/connector-wechat-native/README.md:316]: This is for "universal links" and actually it's "universal link". But it should just for English content. Will remove this

File: packages/connector-wechat-native/README.md [hunk 3]
```diff
@@ -313,7 +313,7 @@ Once WeChat native connector is enabled, you can build and run your app to see i
 
 如果你在「应用已上架」中选择了「否」，你可以在这一步跳过「AppStore 下载地址」的填写。
 
-填写 _Bundle ID_、_测试版本 Bundle ID_ 和 _Universal Links_（实际只需要一个 link 😂）。
+填写 _Bundle ID_、_测试版本 Bundle ID_ 和 _Universal Links_（实际上不需要填写多个 link，只填一个即可）。
```
Comment (xiaoyijun) [inline packages/connector-wechat-native/README.md:316]:
```suggestion
填写 _Bundle ID_、_测试版本 Bundle ID_ 和 _Universal Links_（实际上只需填入同一个 link 即可）。
```

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

File: packages/connector-wechat-native/README.md [hunk 4]
```diff
@@ -235,4 +263,159 @@ Once WeChat native connector is enabled, you can build and run your app to see i
...
 
+## 开始上手
+
+如果你还不知道连接器的概念，或者还没有将本连接器添加至你的「登录体验」，请先参见 [Logto 教程](https://docs.logto.io/zh-cn/docs/tutorials/get-started/enable-social-sign-in)。
+
+在原生应用中，你无法通过 web 进行微信登录：必须跳转至微信 app，并使用他们提供的 SDK 才能完成。
+
+听起来比较吓人？没关系。我们将在这个教程中轻松解决。
+
+
```
Comment (IceHe) [inline packages/connector-wechat-native/README.md:274]: unnecessary two consecutive blank lines
```suggestion

```

**Review by darcyYe — APPROVED** [review]

**Pull request merged**